node/node.go

```diff
@@ -332,27 +324,37 @@ func New(host p2p.Host, consensusObj *consensus.Consensus, chainDBFactory shardc
 
 		// Add Faucet contract to all shards, so that on testnet, we can demo wallet in explorer
 		// TODO (leo): we need to have support of cross-shard tx later so that the token can be transferred from beacon chain shard to other tx shards.
-		if node.isFirstTime {
-			// Setup one time smart contracts
-			//node.AddFaucetContractToPendingTransactions()
-		} else {
-			node.AddContractKeyAndAddress(scFaucet)
-		}
-
-		if node.Consensus.ShardID == 0 {
-			// Contracts only exist in beacon chain
+		if config.Network != config.Mainnet {
 			if node.isFirstTime {
 				// Setup one time smart contracts
-				node.CurrentStakes = make(map[common.Address]*structs.StakeInfo)
-				node.AddStakingContractToPendingTransactions() //This will save the latest information about staked nodes in current staked
+				//node.AddFaucetContractToPendingTransactions()
```

Comment:
remove this comment.

Reply:
reverted, shouldn't be commented if it's not mainnet.